src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -417,7 +419,9 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_REGISTER_PASS_X64(manager, ConvertToInteraction);
     CPU_REGISTER_PASS_X64(manager, ConvertInteractionInt8);
     CPU_REGISTER_PASS_ARM(manager, ConvertReduceMultiAxis);
+#if !defined(OPENVINO_ARCH_ARM64)
```

Comment:
I would prefer to avoid unnessesary explicit defines here, Lets introduce CPU_REGISTER_PASS_ARM32.

Reply:
done